docs/client-options.md

```diff
@@ -118,6 +118,7 @@ third party clients.
 - [Certify](https://github.com/webprofusion/Certify) GUI (.NET, WinForms)
 - [oocx/acme.net](https://github.com/oocx/acme.net) (.NET)
 - [kelunik/acme-client](https://github.com/kelunik/acme-client) (PHP)
+- ZeroSSL: [Portable Perl version](https://zerossl.com/installation.html#portablePerl), [Portable Win32/Win64 binaries](https://github.com/do-know/Crypt-LE/releases).
```

Comment:
Could you confirm whether the reasoning given is good enough to proceed as is or you'd like this to be changed anyway? As I mentioned, it's not a problem to change it at all and I'm sure the shorter version might look better indeed, though I'd probably kept it as "ZeroSSL Win32/Win64 portable", since they are portable :) Thanks.